.github/workflows/release.yml

```diff
@@ -121,6 +121,7 @@ jobs:
         run: ./ossutil --config-file .ossutilconfig sync ./_bin/vela oss://$BUCKET/binary/vela/${{ env.VELA_VERSION }}
       
       - name: sync the latest version file
+        if: !contains(env.VELA_VERSION,'alpha') && !contains(env.VELA_VERSION,'beta')
```

Comment:
If could compare with remote value, it is better. otherwise, 1.3.6 maybe replace the 1.4.0.

Reply:
This is another problem, let's track it here: https://github.com/kubevela/kubevela/issues/3994